public/stylesheets/base.sass

```diff
@@ -2,4 +2,18 @@ body
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif
 
 a
-  color: #00B7FF
\ No newline at end of file
+  color: #00B7FF
```

Comment:
Looove the look of this! Also I think it was a good idea to put the logout button where you did! Super clean, and we can add stuff to that drop down later too if needed! 👍

Reply:
Could we add a drop shadow to the icon? Just to make it a bit more obvious that its a button :)

Reply:
<img width="1680" alt="screen shot 2018-02-25 at 12 50 45 pm" src="https://user-images.githubusercontent.com/22732449/36644027-8c0ea348-1a2a-11e8-8b97-c462e224f362.png">

Just found some issues on mobile. The drop down is clipped. If we found a way to move the user icon beside the navbar drop toggle, that would be dank!

Reply:
Oh no! I looked into fixing this but am having some trouble with sass. Do you know why the syntax we're using looks so different than the syntax I see online?

Reply:
Sass is compiled into CSS when the app starts. Does it look dramatically different?

Reply:
The compiled CSS looks fine to mee!